stable/drupal/OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
```

Comment:
do we need to specify both approves and reviewers? surely an approver supersedes a reviewer?

Reply:
There's a plugin (that I'll get enabled for us) that requests PR reviews by people listed as reviewers. So, we should keep both sections. Does not request reviews from approvers (so it won't bubble up to the top).

Reply:
ah, good to know, thanks!